gulp/publish-tasks/git.js

```diff
@@ -17,7 +17,7 @@ const gitFunctions = {
   clone: (url, folder) => {
     return (cb) => {
       // try to use an SSH URL instead of HTTP(S), so we can push to it gracefully
-      return git.clone(url.replace(/^https?:/, 'ssh:'), { args: folder }, function (err) {
```

Comment:
I think we did not want to commit this change. It is only needed for local run.

Reply:
Aaah yes good point.  It committed because other changes in the file.

I'll update to keep the original replace function.